resources/docker-build_p2p/port_share.sh

```diff
@@ -32,11 +32,25 @@ do
 	--compress=*)    compress=${opt#*=} ;;
 	*)
             if [ ${opt:0:1} = '-' ]; then usage; fi
-            local_port=$opt
+
             ;;
+
     esac
 done
 
+if [ "$1" = '' ]
```

Comment:
since this condition along with the one on line 46 basically call the same action, wouldn't it be possible here to use:
`if [ "$1" = '' ] || [ "$2" = '' ]`
and get rid of the second if statement?